docker/worker/worker.py

```diff
@@ -562,6 +563,14 @@ def _notify_ecosystem_bridge(self, vulnerability):
             push_topic,
             data=json.dumps(osv.vulnerability_to_dict(vulnerability)).encode())
 
+  def _maybe_remove_import_findings(self, vulnerability: osv.Bug):
+    """Remove any stale import findings for a successfully processed Bug,"""
+
+    finding = osv.ImportFinding.get_by_id(vulnerability.id())
+    if finding:
+      logging.info('Removing stale import finding for %s', vulnerability.id())
+      finding.key.delete()
```

Comment:
Just a random thought, will we ever need to query historical finding data? For example, to evaluate how many records the linter helped fix?

Reply:
> will we ever need to query historical finding data?

In my opinion, it's more of a "nice to have" than a strong "need" in terms of requirement. Yes, it's helpful to be able to quantify impact. We can determine that from the logs for as long as we have log retention for...